Makefile

```diff
@@ -0,0 +1,120 @@
+.PHONY: all
+#Run all checks, default when running 'make'
+all: analyze test
+
+#Non phony targets for phars etc.
+vendor: composer.json composer.lock
+	composer install
```

Comment:
Wouldn't it be nice if it would run `composer update` for us too, would it?

```make
# Do install if there's no 'vendor'
vendor/autoload.php:
    composer install

# If composer.lock is older than `composer.json`, do update,
# and touch composer.lock because composer not always does that
composer.lock: composer.json
    composer update && touch composer.lock

# We need both vendor/autoload.php and composer.lock being up to date
.PHONY: vendor
vendor: vendor/autoload.php composer.lock
```